include/swift/AST/Module.h

```diff
@@ -1320,7 +1320,9 @@ class SourceFile final : public FileUnit {
   }
   
   void markDeclWithOpaqueResultTypeAsValidated(ValueDecl *vd);
-  
+
+  ~SourceFile();
```

Comment:
Nitpick: please put this near the constructors. (I think it does need to be public.)

Reply:
Done